movieapp/src/containers/FilterElement/FilterElement.tsx

```diff
@@ -0,0 +1,44 @@
+import { FilterElement } from './FilterElementTypes';
+import { Option } from '../../components/Option/Option';
+import { useDispatch } from 'react-redux';
+import { receiveMoviesFromApiError, sortMoviesById } from '../../actions/actions';
+import { AnyAction } from 'redux';
+
+const movieSortCriterias: Array<FilterElement> = [
+  {
+    id: '1',
+    title: 'Drama',
+    value: 'drama'
+  },
+  {
+    id: '2',
+    title: 'Comedy',
+    value: 'comedy'
+  },
+  {
+    id: '3',
+    title: 'Animation',
+    value: 'animation'
+  }
+];
+
+export function FilterElement(): JSX.Element {
```

Comment:
do not use postfixes naming as `*Element`, `*Array` it's antipattern

Reply:
Fixed